spec/services/images/optimizer_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Images::Optimizer, type: :service do
+  include CloudinaryHelper
+
+  let(:image_url) { "https://i.imgur.com/fKYKgo4.png" }
+
+  it "performs exactly like cl_image_path" do
+    cloudinary_url = cl_image_path(image_url,
+                                   type: "fetch",
+                                   width: 50, height: 50,
+                                   crop: "imagga_scale",
+                                   quality: "auto",
+                                   flags: "progressive",
+                                   fetch_format: "auto",
+                                   sign_url: true)
+    expect(described_class.call(image_url, width: 50, height: 50, crop: "imagga_scale")).to eq(cloudinary_url)
```

Comment:
Are we talking to a stubbed cloudinary object here to get these URL results?

Reply:
This is not a stubbed method. This is the gem provided method that generates a URL. 

This spec is to make sure the combination of provided args and `DEFAULT_CL_OPTIONS` would yield the same output as how we previously use `cl_image_tag` directly.

Reply:
Ah great, wasnt sure if it was making an external call or something to generate the URL.